src/helpers/discord_utils.ts

```diff
@@ -671,3 +671,29 @@ export function sendDebugAlertWebhook(title: string, description: string, color:
         avatar_url: avatarUrl,
     });
 }
+
+export async function sendBookmarkedSongs(bookmarkedSongs: { [userID: string]: Map<string, QueriedSong> }) {
+    for (const [userID, songs] of Object.entries(bookmarkedSongs)) {
+        const allEmbedFields: Array<{ name: string, value: string, inline: boolean }> = [...songs].map((song) => ({
+            name: `"${song[1].originalSongName}" (${song[1].publishDate.getFullYear()}) - ${song[1].artist}`,
+            value: `https://youtu.be/${song[1].youtubeLink}`,
+            inline: false,
+        }));
+
+        const dmChannel = await state.client.getDMChannel(userID);
+        for (const fields of chunkArray(allEmbedFields, 25)) {
+            const embed: Eris.EmbedOptions = {
+                author: {
+                    name: "Kimiqo",
+                    icon_url: KmqImages.READING_BOOK,
+                },
+                title: bold("Bookmarked Songs"),
+                fields,
+                footer: { text: `Played on ${friendlyFormattedDate(new Date())}` },
+            };
+
+            await state.client.createMessage(dmChannel.id, { embeds: [embed] });
```

Comment:
Maybe use pagination instead?

Reply:
I avoided pagination since the user might want to come back to the list later. Since pagination has a timeout they wouldn't be able to access their songs in the future